packages/grafana-ui/src/components/Logs/getLogRowStyles.ts

```diff
@@ -80,6 +80,9 @@ export const getLogRowStyles = stylesFactory((theme: GrafanaTheme, logLevel?: Lo
           }
         }
       }
+      td:last-child {
+        width: 100%;
```

Comment:
Do we need this now? If we do not set it to 100% does it mean the last column won't take the rest of the table?

Reply:
It has to be there, as for `Unique Labels` we are setting `max-width: 22em` and sometimes, if the label is very short, and log message is also short, html table creates wider `Unique Labels` column than necessary. This prevents that.